modules/imaging_qc/README.md

```diff
@@ -0,0 +1,34 @@
+## Imaging Quality Control Module
+
+#### Purpose
+The purpose of this module is to verify and help ensure imaging data has been uploaded into LORIS and reviewed for Quality Control (QC). It provides a queryable interface to check at a glance the completeness of imaging data collection for a study, and the status of QC review processes on them.  
+
+Note: this module is not the module in which visual/radiological QC is performed on scans, just the module in which QC assessments are *reviewed*. Links are included in this module so the user can jump to the Imaging Browser to mark QC assessments on images and visits.
+
+#### Intended Users
+This module is used by study administrators, imaging scientists, analysts, coordinators, and clinicians.
+
+#### Scope
+The Imaging QC module allows you to view specific data on uploaded image files, with many additional filtering options. The resulting table has a column for QC status, so you can see whether the upload has passed QC or not. The table provides a link to each file. You can download the table in `.csv` format. 
+
+Note: To QC a scan, the user must click the link leading to the Imaging Browser, and enter the QC details in that module.
+
+#### Configuration caveat 
+
+This module has a key dependency - the `mri_parameter_form` instrument must installed. 
+If not, this module will not and error message will indicate that `Table 'LORIS.mri_parameter_form` is missing. 
```

Comment:
Look like this line need some work. This module will not what?

Reply:
Also `will not load an error message`, not "and"

Reply:
"and an"

Reply:
doh.  thanks guys.